main.go

```diff
@@ -6,6 +6,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/hashicorp/go-multierror"
```

Comment:
We might use https://github.com/efficientgo/core/tree/main/merrors here instead. 🙂

Reply:
👍🏽 love to see it

Fixed.

Reply:
Yes please!